sklearn/metrics/pairwise.py

```diff
@@ -518,6 +519,10 @@ def manhattan_distances(X, Y=None, sum_over_features=True,
     array([[ 1.,  1.],
            [ 1.,  1.]]...)
     """
+    if size_threshold is not None:
+        warnings.warn('Use of the "size_threshold" is deprecated '
```

Comment:
can you please explicitly mention the version in which it is deprecated (0.19) and the one in which it will be removed (0.21)